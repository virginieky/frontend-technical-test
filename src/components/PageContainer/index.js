import styled from 'styled-components';

const PageContainer = styled.div`
  height: 100vh;
  padding-top: 70px;
  background-color: #f5f5f5;
`;

export default PageContainer;
