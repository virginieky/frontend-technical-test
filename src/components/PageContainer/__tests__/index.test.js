import React from 'react';
import { render } from '@testing-library/react';
import PageContainer from '../index';

describe('components | PageContainer', () => {
  it('should match the snapshot', () => {
    const { asFragment } = render(<PageContainer />);

    expect(asFragment()).toMatchSnapshot();
  });
});
