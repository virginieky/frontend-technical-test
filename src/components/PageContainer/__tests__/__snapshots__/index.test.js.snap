// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | PageContainer should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="PageContainer-sc-1gdmd4z-0 ewlSdy"
  />
</DocumentFragment>
`;
